### PR TITLE
Various CI fixes and improvements

### PR DESCRIPTION
**Pull request by Luthaf**
Concerning the PDB test: on my machine, removing just the test that creates 100000+ atoms bring the run time under valgrind fromp 400s to 120s. Removing track-origins bring it to 60s.

### CODE REVIEW

File: external/CMakeLists.txt [hunk 1]
```diff
@@ -60,6 +60,7 @@ set(EXTERNAL_INCLUDES "")
 # ==========
 # fmtlib: https://github.com/chemfiles/fmt
 # ==========
+set(FMT_USE_CPP14 OFF CACHE bool "Enable the addition of C++14 compiler flags.")
```
Comment (frodofine) [inline external/CMakeLists.txt]: Fair enough, that got annoying.

File: appveyor.yml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 version: "{build}"
-
 os: Visual Studio 2015
 
+branches:
```
Comment (frodofine) [inline appveyor.yml]: Will this prevent the double builds when requesting a PR?

File: appveyor.yml [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
+  only:
```
Comment (frodofine) [inline appveyor.yml]: It is good to see the build status of some branches when developing them. IE while implementing the GRO parser I could see if CI worked for all commits along the way and I could fix issues as I went.  Maybe I'll just do development in my forked and submit PRs from there :smile_cat:

File: scripts/ci/build-docs.sh [hunk 4]
```diff
@@ -22,7 +22,9 @@ rm -rf gh-pages/deployed*
 cmake -DCHFL_BUILD_DOCUMENTATION=ON .
 cmake --build . --target doc_html
 rm -rf doc/html/.doctrees/ doc/html/.buildinfo
-rm -rf doc/html/_static/bootswatch-* doc/html/_static/bootstrap-2.3.2/
+rm -rf doc/html/_static/bootswatch-2.3.2/ doc/html/_static/bootstrap-2.3.2/
```
Comment (frodofine) [inline scripts/ci/build-docs.sh]: I'm not sure why this is necessary at first glance. Mind explaining it to me?

File: tests/CMakeLists.txt [hunk 5]
```diff
@@ -38,8 +38,7 @@ set_property(CACHE CHFL_TEST_RUNNER PROPERTY STRINGS none valgrind wine node)
 if(${CHFL_TEST_RUNNER} STREQUAL "valgrind")
     set(
         RUNNER_COMMAND
-        "valgrind" "--leak-check=full" "--dsymutil=yes" "--track-origins=yes"
-        "--error-exitcode=125"
+        "valgrind" "--leak-check=full" "--dsymutil=yes" "--error-exitcode=125"
```
Comment (frodofine) [inline tests/CMakeLists.txt]: Reduced PDB time from 630s to 76s on Travis :+1:

File: tests/formats/pdb.cpp [hunk 6]
```diff
@@ -343,17 +344,19 @@ TEST_CASE("PDB files with big values") {
     }
 
     SECTION("CONNECT with too many atoms") {
-        auto tmpfile = NamedTempPath(".pdb");
-
-        auto frame = Frame();
-        for(size_t i=0; i<110000; i++) {
-            frame.add_atom(Atom("A"), {0.0, 0.0, 0.0});
+        if (!is_valgrind_and_travis()) {
```
Comment (frodofine) [inline tests/formats/pdb.cpp]: Or maybe this reduced the timings!

The same should be done with the GRO format to ensure that test finishes in time as well.

File: appveyor.yml (hunk 2 above)
Comment (Luthaf) [reply appveyor.yml]: Yes, that's the idea!

File: appveyor.yml (hunk 3 above)
Comment (Luthaf) [reply appveyor.yml]: > Maybe I'll just do development in my forked and submit PRs from there

Yes, that's what I do. I think you can enable Travis for you fork too

File: scripts/ci/build-docs.sh (hunk 4 above)
Comment (Luthaf) [reply scripts/ci/build-docs.sh]: The documentation (http://chemfiles.org/chemfiles/latest/) is missing `boostrap.min.css` which is fetched from `doc/html/_static/bootswatch-3.7.3/flatly/bootstrap.min.css`

File: tests/formats/pdb.cpp (hunk 6 above)
Comment (Luthaf) [reply tests/formats/pdb.cpp]: Will do! I forgot about GRO

Comment (Luthaf) [reply tests/formats/pdb.cpp]: Done and rebased!

Comment (frodofine) [reply tests/formats/pdb.cpp]: Great!

**Pull request merged**